sonic_data_client/virtual_db.go

```diff
@@ -34,6 +34,11 @@ var (
 	// Queue name to oid map in COUNTERS table of COUNTERS_DB
 	countersQueueNameMap = make(map[string]string)
 
+	// Alias translation: from external name to internal name
+	e2i_aliasMap = make(map[string]string)
```

Comment:
>e2i_aliasMap [](start = 1, length = 12)

change to name2aliasMap and alias2nameMap.

Reply:
there is no internal/external name concept, let use name to refer to sonic interface name and alias to refer vendor port name.

---
In reply to: [213116801](https://github.com/Azure/sonic-telemetry/pull/13#discussion_r213116801) [](ancestors = 213116801)

Reply:
Variable names are renamed. Please review.